MAUI/DataGrid/Styling.md

```diff
@@ -9,15 +9,19 @@ documentation: ug
 
 # Styling in .NET MAUI DataGrid (SfDataGrid)
 
-The data grid applies style for all of its elements by setting desired values to the style properties in SfDataGrid.DefaultStyle.
+The [DataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) applies style for all of its elements by setting desired values to the style properties in [SfDataGrid.DefaultStyle] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_DefaultStyleProperty).
```

Comment:
The [DataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) applies style for all of its elements by setting the desired values to the style properties in [SfDataGrid.DefaultStyle] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_DefaultStyleProperty).